seeders/20180219074913-ActionsSeed.js

```diff
@@ -6,8 +6,58 @@ module.exports = {
     return queryInterface.bulkInsert('Actions', [
 
       { name: "HTML boilerplate", url: "https://www.sitepoint.com/introduction-html5-boilerplate/" , criterionId: 10000, createdAt: new Date(), updatedAt: new Date(), id: 10000 },
-      { name: "CSS", url: "https://github.com/twbs/bootstrap" , criterionId: 10000, createdAt: new Date(), updatedAt: new Date(), id: 10002 },
```

Comment:
Why is bootstrap relevant to this first criteria.  The criteria is "Can write the basic HTML boilerplate from memory".  The key word here is **basic**.  Basic is all the stuff needed to get started in an HTML file. This includes things like this...

    - `Doctype`
    - `html tag`
    - `head tag`
    - `meta tags`
    - `styles`
    - `scripts`
    - `body tag`

Reply:
Thanks for the feedback.